cdap-ui/app/cdap/components/ResourceCenterEntity/index.js

```diff
@@ -27,13 +36,16 @@ export default function ResourceCenterEntity({className, iconClassName, title, d
             <div className={classnames("entity-image", iconClassName)}/> :
             <div className="entity-image empty"></div>
         }
-        <button
-          className={classnames("btn btn-primary")}
-          onClick={onClick}
-          disabled={disabled}
-        >
-          {actionLabel}
-        </button>
+        {
+          actionLink ?
+            (
+              <a href = {actionLink}>
+                {button}
```

Comment:
It's not just the label right, we also want to wrap the entire button in the anchor tag?